frontend/test/metabase/scenarios/onboarding/setup/user_settings.cy.spec.js

```diff
@@ -214,4 +214,56 @@ describe("user > settings", () => {
       cy.findByLabelText("Email").should("not.exist");
     });
   });
+
+  describe("when user is authenticated via JWT", () => {
+    beforeEach(() => {
+      cy.server();
+      cy.route(
+        "GET",
+        "/api/user/current",
+        Object.assign({}, CURRENT_USER, {
+          sso_source: "jwt",
+        }),
+      ).as("getUser");
+
+      cy.visit("/account/profile");
+      cy.wait("@getUser");
+    });
+
+    it("should hide change password tab", () => {
+      cy.findByText("Password").should("not.exist");
+    });
+
+    it("should hide first name, last name, and email input (metabase#23298)", () => {
+      cy.findByLabelText("First name").should("not.exist");
+      cy.findByLabelText("Last name").should("not.exist");
+      cy.findByLabelText("Email").should("not.exist");
+    });
+  });
+
+  describe("when user is authenticated via SAML", () => {
+    beforeEach(() => {
+      cy.server();
+      cy.route(
```

Comment:
`cy.route` is deprecated. Can we change it to [cy.intercept()](https://docs.cypress.io/api/commands/intercept)? There are many examples in our tests.

Reply:
Good call. I just blindly copied from the adjacent tests. 😅